repository/database/verification.py

```diff
@@ -37,6 +48,55 @@ def get_mail(self, fallback_domain: str) -> str:
 
         return f"{self.login}@{fallback_domain}"  # fallback
 
+    @classmethod
+    def get_user(cls, login: str, status: int = 2) -> Optional[ValidPersonDB]:
+        """"Finds login from database"""
+        user = (
+            session.query(ValidPersonDB)
+            .filter(ValidPersonDB.login == login, ValidPersonDB.status == status)
+            .one_or_none()
+        )
+        return user
+
+    @classmethod
+    def add_user(cls, login: str, year: str, status: int = 1) -> ValidPersonDB:
+        """Add user to database"""
+        person = ValidPersonDB(login=login, year=year, status=status)
+        session.add(person)
+        session.commit()
+        return person
+
+    @classmethod
+    def get_user_by_login(cls, login: str) -> Optional[ValidPersonDB]:
+        """Finds login from DB (without status check)"""
+        user = (
+            session.query(ValidPersonDB).filter(ValidPersonDB.login == login).one_or_none()
+        )
+
+        return user
+
+    def save_verified(self, login: str, discord_id: str) -> None:
+        """"Inserts login with discord name into database"""
+        session.add(PermitDB(login=login, discord_ID=discord_id))
```

Comment:
You don't need to pass login. It is already in the self object.
```suggestion
        session.add(PermitDB(login=self.login, discord_ID=discord_id))
```